<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEdit</class>
 <widget class="BaseDlg" name="DialogEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DialogEdit</string>
  </property>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="yesButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>yes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>42</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>pos_col</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pos">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>42</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>pos_row</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rack">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
